ui/threejs/mesh.js

```diff
@@ -13,7 +13,7 @@ import {
   Vector2,
   Vector3,
   VertexColors,
-} from 'three';
+} from 'three/build/three.module.js';
```

Comment:
Why the switch to more explicit imports in these?

Reply:
Good catch -- I did this initially because there was an issue with importing them into node, but that was factored out.

I'll revert these.